layers/meta-balena-beaglebone/recipes-core/images/balena-image-flasher.bbappend

```diff
@@ -1,5 +1,8 @@
 include balena-image.inc
 
 BALENA_BOOT_PARTITION_FILES:append:beaglebone = " uEnv.txt_internal:"
+
 IMAGE_INSTALL:append:beaglebone = " bb-org-overlays"
 IMAGE_INSTALL:append:beaglebone-ai64 = " mmc-utils vxd-dec-fw"
+IMAGE_INSTALL:append:beagleplay = " mmc-utils"
+IMAGE_INSTALL:append:beagleplay-k3r5 = " mmc-utils"
```

Comment:
To be removed, we do not use that machine type

Reply:
Not sure I follow, @floion - I think we need the mmc-utils package because it provides some tools needed by the flasher hook scripts. This is the original commit that added it: https://github.com/delhiryder/balena-beaglebone/commit/f6da0e1760d7cffb33c23488091cfc7d63815ede

Reply:
I think I understand now. You want me to remove the `mmc-utils` for the `beagleplay-k3r5` MACHINE. Will do this and update the PR after verifying locally.

Reply:
OK

Reply:
Done.